include/ppx/application.h

```diff
@@ -527,6 +527,11 @@ class Application
     // Initializes standard knobs
     void InitStandardKnobs();
 
+    // This is used by derived class to overwrite standard knobs default value
+    virtual void SetupStandardKnobsDefaultValue()
```

Comment:
Maybe `virtual void SetupStandardKnobsDefaultValue(StandardKnobsDefaultValue&)`?

Reply:
i think the name was confusing, changed the name to `OverwriteStandardKnobsDefaultValue` to be clear that it is only for overwriting the default values

Reply:
What I mean is passing down a reference instead of using `protected:`, the derived class should not touch the default value outside of this function anyway. Similar to `Config(ppx::ApplicationSettings&)`

Reply:
Context: "Make classes' data members private, unless they are constants." [cppguide#Access_Control](https://google.github.io/styleguide/cppguide.html#Access_Control) (also in go/cstyle)
BigWheels does not follow all the google style guide for reasons, but I'd keep to the style guide when reasonable.

Reply:
Thanks for the link. Make sense, changed as suggested